ethportal-peertest/src/lib.rs

```diff
@@ -2,6 +2,32 @@ pub mod cli;
 pub mod events;
 pub mod jsonrpc;
 
-pub use jsonrpc::{test_jsonrpc_endpoints_over_http, test_jsonrpc_endpoints_over_ipc};
-
 pub use cli::PeertestConfig;
+pub use jsonrpc::{get_enode, test_jsonrpc_endpoints_over_http, test_jsonrpc_endpoints_over_ipc};
+
+use std::sync::Arc;
+
+use trin_core::cli::TrinConfig;
+use trin_core::jsonrpc::service::JsonRpcExiter;
+
+pub async fn launch_buddy_node() -> (String, Arc<JsonRpcExiter>) {
```

Comment:
I think we can make this method more general and add `discovery_port` and `web3_ipc_path` as method arguments. This way, we can use it to spin multiple nodes.